formats/json-io/build.gradle.kts

```diff
@@ -33,12 +33,8 @@ kotlin {
 
 project.configureJava9ModuleInfo()
 
-tasks.named<DokkaTaskPartial>("dokkaHtmlPartial") {
-    dokkaSourceSets {
-        configureEach {
-            externalDocumentationLink {
-                url.set(URI("https://kotlin.github.io/kotlinx-io/").toURL())
-            }
-        }
+dokka.dokkaSourceSets.configureEach {
+    externalDocumentationLinks.register("kotlinx-io") {
+        url("https://kotlinlang.org/api/kotlinx-io/")
```

Comment:
JIC: old URL is not accessible anymore

Reply:
Hm, that looks strange, local build works fine on my side.
I can make it use the local package list (downloaded), as it was before for okio. The only case when the package list will become outdated is when a package is added or removed, so probably it should not be a big deal here to just use the local package list.
WDYT?

Reply:
I've added local package lists for both kotlinx-io and okio, so that the build will be less dependent on network failures.

JIC: Dokka by default still relies on downloading the package list of the stdlib.